docs/Home.md

```diff
@@ -0,0 +1,87 @@
+# Google Kubernetes Engine Plugin Wiki
+
+The [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) Plugin allows you to publish deployments built within Jenkins to your Kubernetes clusters running within GKE.
+
+_Plugin Information_
+
+View Google Kubernetes Engine [on the plugin site](https://plugins.jenkins.io/google-kubernetes-engine) for more information.
+
+## Usage
+
+### Enable Required APIs
+
+1. Export project.
+    ```bash
+    export PROJECT=$(gcloud info --format='value(config.project)')
+    ```
+2. Enable GKE, Google Compute Engine (GCE), Service Management, and Cloud Resource Manager APIs.
+    ```bash
+    gcloud services enable --project $PROJECT compute.googleapis.com container.googleapis.com servicemanagement.googleapis.com cloudresourcemanager.googleapis.com
+    ```
+
+### IAM Credentials
+
+1. Create a service account using the Google Cloud SDK.
+    ```bash
+    gcloud iam service-accounts create jenkins-gke
+    ```
+1. Add the KubernetesEngineAdmin, serviceAccountUser, and computeNetworkViewer roles to the service account:
+    ```bash
+    export SA_EMAIL=$(gcloud iam service-accounts list --filter="name:jenkins-gke" --format='value(email)')
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/iam.serviceAccountUser $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.clusterAdmin $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.admin $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/compute.networkViewer $PROJECT
+    ```
+1. Download a JSON Service Account key for your newly created service account. Take note of where the file was created, you will upload it to Jenkins in a subsequent step.
+    ```bash
+    gcloud iam service-accounts keys create ~/jenkins-gke-key.json --iam-account jenkins-gke@${PROJECT}.iam.gserviceaccount.com
```

Comment:
Just noticed that we have an existing variable with the SA email address: $SA_EMAIL. Should probably use it here too.